packages/app-content-pages/src/screens/About/components/OurMission.js

```diff
@@ -1,14 +1,52 @@
-import { Anchor, Heading, Paragraph } from 'grommet'
+import {
+  Anchor,
+  Box,
+  Button,
+  Heading,
+  Image,
+  Paragraph,
+  Text,
+  Tip
+} from 'grommet'
+import { CircleInformation } from 'grommet-icons'
 import { Trans, useTranslation } from 'next-i18next'
-import Link from 'next/link'
+import styled from 'styled-components'
```

Comment:
This component will eventually be a component library, lib-content-pages, so I'm removing Next Links for now.